packages/rum-core/src/rawRumEvent.types.ts

```diff
@@ -181,7 +181,7 @@ export type RawRumEvent =
   | RawRumLongTaskEvent
   | RawRumActionEvent
 
-export interface RumContext {
+export interface RumContext extends Context {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ we need to extend `Context` here just to work around the `Index signature for type 'string' is missing in type ...` error, but `RumContext` is never actually used with an index, except when using `combine`.

Maybe we could remove the `extends` here, and cast the object as `RumContext & Context` where `RumContext` is used with `combine`. WDYT?

I know this is a bit confusing, and there is a [long standing TS issue](
https://github.com/microsoft/TypeScript/issues/15300) related to that.